agent/agent.go

```diff
@@ -66,6 +66,7 @@ var (
 	healthCheckPath      = flag.String("health-check-path", "/", "Path on backend host to issue health checks against.  Defaults to the root.")
 	healthCheckFreq      = flag.Int("health-check-interval-seconds", 0, "Wait time in seconds between health checks.  Set to zero to disable health checks.  Checks disabled by default.")
 	healthCheckUnhealthy = flag.Int("health-check-unhealthy-threshold", 2, "A so-far healthy backend will be marked unhealthy after this many consecutive failures. The minimum value is 1.")
+        disableGCEVM         = flag.Bool("disable-gce-vm", false, "Disable the agent from adding a GCE VM header.")
```

Comment:
Can you add '-header' to the name of the flag so it's clearer what is being disabled?

Reply:
Done